1st. peer to peer review

lmunozm1702 opened the pull request.

lmunozm1702 committed:
1st. commit

sebalp100 commented on src/index.js:
```diff
@@ -0,0 +1,30 @@
+const clearAll = document.querySelector('#clear-all');
+clearAll.addEventListener('click', () => {
+  myList.removeTask('COMPLETED');
+});
```
I think it would be better to have all your declarations at the top of the script in accordance with JS best practices: https://www.w3schools.com/js/js_best_practices.asp

I think this would make your code look more professional and organized.

sebalp100 commented on README.md:
```diff
@@ -0,0 +1,84 @@
+
+remove all of the completed task, using the Clear all link at bottom of the app.
+
+### Deployment
+
+Not available yet.
+
```
I think it would be better to add a live version demo of your project in the readme. You can use GitHub pages and it would look more professional since your visitors would be able to see your app without having to download your code.

sebalp100 reviewed (commented):
# STATUS: REQUIRES CHANGES ♻️ 

![dog](https://user-images.githubusercontent.com/111214189/200394863-7ac4b744-1dc2-4374-9397-cefe6b51c422.gif)

Great job so far @lmunozm1702 , your code is looking great! 🥳
However, I think there are some issues that can be fixed to make your code better.

Highlights: 

✔️ Used GitHub Flow
✔️ No linter errors


### Required changes ♻️ 

Please make sure to use a descriptive PR so reviewers can know what was implemented for this milestone. 

_Check the comments under the review_

sebalp100 reviewed (changes requested):
# STATUS: REQUIRES CHANGES ♻️ 

![dog](https://user-images.githubusercontent.com/111214189/200394863-7ac4b744-1dc2-4374-9397-cefe6b51c422.gif)

Great job so far @lmunozm1702 , your code is looking great! 🥳
However, I think there are some issues that can be fixed to make your code better.

Highlights: 

✔️ Used GitHub Flow
✔️ No linter errors


### Required changes ♻️ 

Please make sure to use a descriptive PR so reviewers can know what was implemented for this milestone. 

_Check the comments under the review_

lmunozm1702 committed:
Fix code review issues

Updates README.md and index.js to solve:

- have all your declarations at the top of the script in accordance with JS best practices
- add a live version demo of your project in the readme.

lmunozm1702 commented:
Hi @sebalp100 I fix both issues, please help with a re-review of the code.

sebalp100 reviewed (approved):
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

![ok](https://user-images.githubusercontent.com/111214189/200399308-b9ce09df-5ae1-4836-b70e-f86a08c7e984.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
